test/e2e/test_cluster_configs/cloud_controller_manager.json

```diff
@@ -1,21 +1,20 @@
 {
+    "env": {
+        "GINKGO_SKIP": "should be able to attach azure file"
```

Comment:
Skipping this test case because azurefile-csi-driver isn't supported in Windows clusters yet. https://github.com/chewong/aks-engine/tree/windows-csi will address this issue and I will open a second PR after this PR is merged.